husky_components/data-navigation/main.js

```diff
@@ -237,6 +306,18 @@ define([
             this.sandbox.on(CLEAR_CACHE.call(this), function() {
                 this.cache.deleteAll();
             }.bind(this));
+
+            this.sandbox.on('husky.search.data-navigation', function(searchTerm) {
+                this.searchTerm = searchTerm;
```

Comment:
reading from a input which is created by a component is dirty, then we should create an event and return this over a callback! but then `getCurrentUrl` has to call this event everytimes ...